posthog/temporal/tests/test_squash_person_overrides_workflow.py

```diff
@@ -573,6 +574,32 @@ async def test_squash_events_partition(activity_environment, person_overrides_da
     assert_events_have_been_overriden(events_to_override, person_overrides_data)
 
 
+@pytest.mark.django_db
+@pytest.mark.asyncio
+async def test_squash_events_partition_dry_run(activity_environment, person_overrides_data, events_to_override):
+    """Test events are not squashed by running squash_events_partition with dry_run=True."""
+    dictionary_name = "exciting_dictionary"
+
+    query_inputs = QueryInputs(
+        partition_ids=["202001"],
+        database=settings.CLICKHOUSE_DATABASE,
+        dictionary_name=dictionary_name,
+        user=settings.CLICKHOUSE_USER,
+        password=settings.CLICKHOUSE_PASSWORD,
+        cluster_name=settings.CLICKHOUSE_CLUSTER,
+        _latest_merged_at=OVERRIDES_MERGED_AT.isoformat(),
+        dry_run=True,
+    )
+    await activity_environment.run(prepare_dictionary, query_inputs)
+
+    await activity_environment.run(squash_events_partition, query_inputs)
+
+    await activity_environment.run(drop_dictionary, query_inputs)
+
+    with pytest.raises(AssertionError):
+        assert_events_have_been_overriden(events_to_override, person_overrides_data)
```

Comment:
I'd suggest we assert that they are the same instead - checking that we raised an `AssertionError` feels odd

Reply:
Yeah. I agree. I was lazy here.